content/00.front-matter.md

```diff
@@ -37,13 +37,13 @@ on {{manubot.date}}.
     {% if author.orcid is defined -%}
     [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/{{author.orcid}})
     {%- endif %}
-    {{author.name}}<sup>{% for number in author.affiliation_numbers %}2.{{number}}{%- if not loop.last -%},{%- endif -%}{% endfor %}♠{%- if author.symbol_str is defined -%},{{author.symbol_str}}{%- endif -%}</sup>
+    {{author.name}}<sup>{% for number in author.affiliation_numbers %}2.{{number}}{%- if not loop.last -%},{%- endif -%}{% endfor %}⚄{%- if author.symbol_str is defined -%},{{author.symbol_str}}{%- endif -%}</sup>
     {%- if not loop.last -%}, {%- endif -%}
   {%- endif -%}
 {% endfor %},
 [The Version 1.0 Deep Review Authors](#version-1.0-authors)
 
-<sup>♠</sup> --- Author order for version 2.0 is currently randomized [with each new build](https://github.com/greenelab/deep-review/pull/997).<br>
+<sup>⚄</sup> --- Author order for version 2.0 is currently randomized [with each new build](https://github.com/greenelab/deep-review/pull/997).<br>
```

Comment:
@cgreene let's see how a dice looks. There is also 🎲, but it's part of Unicode 6.0 from 2010 so may have less support.

Reply:
I tried the new dice and it had no dice. Just a square. Sounds good to me 👍